src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -12,7 +12,7 @@ namespace System.ComponentModel
     /// <summary>
     ///     Represents a collection of attributes.
     /// </summary>
-    public class AttributeCollection : ICollection
+    public class AttributeCollection : ICollection, IEnumerable
```

Comment:
I thought about that... and I kept it there since we are also explicitly implementing those members...

I am fine to remove it if you think it does not add any value to have it there.